src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -267,6 +292,11 @@ private void UpdateCurrentConfiguration()
             }
             catch (Exception ex)
             {
+                long elapsedTime = (long)_timeProvider.GetElapsedTime(startTimestamp).TotalMilliseconds;
+                _telemetryClient.LogOperationDuration(
```

Comment:
The interface would be better if the telemetry client took the exception instead.